### DISCUSSION TITLE
#2 枣庄多线1 无法认证

### DISCUSSION THREAD

**Discussion by eya46** [natfrp/wiki#279] [使用问题] [answered]
认证页面显示：抱歉，请联系机房域名过白

**Comment by eya46** [comment]
http://cn-zz-bgp-1.natfrp.cloud
其余端口和80端口访问也这样，https无法访问

**Comment by fakeboboliu** [comment, marked as answer]
请尝试使用 ip，即 https://43.249.193.233:端口 的形式

**Reply by eya46** [reply]
也是不行的，http和https都试了

**Reply by fakeboboliu** [reply]
我们测试并未发现问题，请重试并检查客户端日志是否存在不正常日志。

**Reply by eya46** [reply]
端口****，很久没管过了，但是连接正常，并且开了自动更新
不知道是哪方面出的问题
https://43.249.193.233:****
http://43.249.193.233:****
http://cn-zz-bgp-1.natfrp.cloud:****
https://cn-zz-bgp-1.natfrp.cloud:****
四种均不行

**Reply by fakeboboliu** [reply]
> 端口22750，很久没管过了，但是连接正常，并且开了自动更新

服务端 debug 信息显示您的客户端版本为 0.42.0-sakura-2.1，请考虑尝试更新

**Reply by eya46** [reply]
ok，等会去看看
